### ISSUE TITLE
Could you please provide enhancement data files and config files?

### ISSUE THREAD

**Issue by hbbOooO** [HenryJunW/TAG#2]
Hello,

I try to run the code and train on TextVQA dataset. But I cannot achieve the accuracy like in the paper.
I think some important details may be mistaken in my experiments, such as the config of training on TextVQA dataset. So, 

1. Could you provide the config file about training TAP model on TextVQA dataset? Here is only config file on  STVQA dataset.
2. Could you please provide the following files: "st_vqaaug_imdb_train_ocr_en_largest_ocr_bbox_0_4.npy" and "aug_imdb_train_ocr_en_largest_ocr_bbox_0_4.npy". I am not sure if I have generated the correct enhancement data.

**Comment by HenryJunW** [comment]
Thanks for your interest in our work. First I would like to apologize that I replied late. 

1. We've uploaded the files here, https://drive.google.com/file/d/1RY1mXBzqHvPpH47XbjAKdajiOPzB_6Ej/view?usp=sharing for textvqa_tap_pretrain_w_tag_stvqa.yaml and https://drive.google.com/file/d/1QUi9uBc8zZUvYWnbwIq93VnCM6rJdU5Z/view?usp=sharing for textvqa_tap_refine_w_tag_stvqa.yaml. 

2. We've uploaded the files here, https://drive.google.com/file/d/1gwDXXo6LYCSaCo1B7r3fbu7j504Vw-Xj/view?usp=share_link for st_vqaaug_imdb_train_ocr_en_largest_ocr_bbox_0_4.npy, and https://drive.google.com/file/d/1ys-0RSQlRhJNVKTTvOuW1ky6qsEuVl_V/view?usp=share_link for aug_imdb_train_ocr_en_largest_ocr_bbox_0_4.npy.

**Comment by assjdshisio** [comment]
could you please provide the ckpt file of the best model on TextVQA(the corresponding config is textvqa_tap_pretrain_w_tag_stvqa.yaml) ?

**Comment by HenryJunW** [comment]
Somehow I missed the question. Here is the best ckpt file for the textvqa_tap_pretrain_w_tag_stvqa.yaml, https://drive.google.com/file/d/1Lvzf6Z1R2Le3Rpd2DDeEYlsSNfNPAHrN/view?usp=share_link. Note that you need fine-tuning from the released pre-training using textvqa_tap_refine_w_tag_stvqa.yaml.
